.github/workflows/terraform-workflow.yml

```
@@ -157,6 +157,12 @@ on:
         description: >
           Encrypted keystore data used for SAML handshake between Okta and Spinnaker
         required: false
+      basic-auth-user:
```

```suggestion
      chart-auth-user:
```
`basic-auth-user` is generic and the purpose of this param is specifically meant for the chart.